src/renderer/grid_renderer.rs

```diff
@@ -103,7 +103,7 @@ impl GridRenderer {
                 .set_color(style.background(&self.default_style.colors).to_color());
         }
 
-        if self.paint.color() == Color::BLACK {
+        if self.paint.color() == self.get_default_background().with_a(255) {
```

Comment:
This causes improper rendering of font shadows. If you look closely on your line numbers you can see the issue, removing it fixes this.

Reply:
I don't see it. And this is needed for transparency.